data/datasets/PASCALVOC/test_PASCALVOCDataset.py

```diff
@@ -16,3 +17,26 @@ def test_raw_dataset():
         objects = element['objects']
         assert type(objects) == dict, f"{type(objects)=}"
         assert set(objects.keys()) == set(['bbox', 'is_difficult', 'is_truncated', 'label', 'pose']), f"{set(objects.keys())=}"
+
+def test_PASCALVOC_dataset():
+    dataset = PASCALVOCDataset(purpose='training', task='object_detection')
+    assert(len(dataset) == 2501)
+    example = dataset.get_example()
+    assert(type(example) == tuple)
+    assert(len(example) == 2)
+
+    assert(len(example[0].get_shape()) == 3)
+    assert(example[0].get_shape()[0] == 480)
```

Comment:
just do `example[0].shape[0:3] == (480, 389, 3)` or just remove the `[0:3]` if you are sure that the shape is a 3-tuple.